Sources/LanguageServerProtocolJSONRPC/JSONRPCConnection.swift

```diff
@@ -172,6 +172,12 @@ public final class JSONRPCConnection {
     }
     return ready
   }
+  
+  private func send(_ message: JSONRPCMessage, async: Bool = true) {
```

Comment:
We should test the new `async == false` behaviour. We can underscore and make this public:

```swift
/// *Public for testing*
public func _send(...)
```

and add a test in `Tests/LanguageServerProtocolJSONRPCTests/ConnectionTests.swift` that uses this.  There are other tests that do similar things like sending raw data.

Reply:
done